### ISSUE TITLE
A11y_.NETCore_ASP.NETCore_BlazorWasmHome_NonTextContrast: Color contrast ratio between selected tab and non selected tab is 2.238:1 which is less than 3:1.

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspnetcore#49790]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Github Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopWeb; #BM_.NETCore_Web_Jul23; #.NET Core; #WCAG1.4.11; #Win11; #NonTextContrast; #ChromiumEdge; #A11yWCAG2.1; #A11ypreview;

### Environment Details:
App name: .NET Core
URL: http://localhost:5268/
Chromium Edge: Version 116.0.1938.29 (Official build) dev (64-bit)

### Repro Steps:

1. Open command prompt and enter command "mkdir foldername" e.g. "mkdir BlazorWasm".
2. Move to the created directory using command "cd foldername".
3. Run command "dotnet new blazorwasm" and hit ENTER key.
4. Run command "dotnet run".
5. Copy the First URL link from Command prompt and paste it in URL section of Browser and hit ENTER key.
6. TAB to "Home" and hit ENTER key. Observe the issue.

### Actual Result:
Color contrast ratio between selected tab and non selected tab is 2.238:1 which is less than 3:1.

### Expected Result:
Color contrast ratio between selected tab and non selected tab should be greater than or equal 3:1.

### User Impact:
Low vision users will not be able to differentiate between selected tab and non selected tab.

### Attachment:
![49790_A11y_ NETCore_ASP NETCore_BlazorWasmHome_NonTextContrast](https://github.com/dotnet/aspnetcore/assets/90901519/4987a12c-b4fb-4df4-80db-0f1a2cb1398d)

**Comment by merriemcgaw** [comment]
@mkArtakMSFT @SteveSandersonMS one more accessibility bug in the templates for you. Can we have someone look at this for RC2?

**Comment by mkArtakMSFT** [comment]
#Closed;

**Comment by merriemcgaw** [comment]
@PoojaNamde can you please validate this issue and confirm we are good to close it?

**Comment by PoojaNamde** [comment]
#Reopened; #Regressed:09-15-23;

@merriemcgaw Verified this issue in the environment http://localhost:5227/, but issue is still able to repro. hence reactivating the bug.

![image](https://github.com/dotnet/aspnetcore/assets/90901519/62254532-c641-4b89-9e40-85ec9a1b911c)

**Comment by mkArtakMSFT** [comment]
How did you validate this? Was there a new RC2 build to use? Because the screenshot you've posted doesn't have the fix that was merged.

**Comment by mkArtakMSFT** [comment]
@merriemcgaw I recommend vendors to wait till the proper RC2 builds start to come out after the CC date. So on Tuesday there should be a build to use.

**Comment by PoojaNamde** [comment]
@mkArtakMSFT 

We are not able to find the .NET RC 2 build details to download. Could you please us to get those details so that we can verify this issue.

**Comment by PoojaNamde** [comment]
We are not able to find the .NET RC 2 build details to download. Could you please us to get those details so that we can verify this issue.

**Comment by merriemcgaw** [comment]
We should have a build early next week. I will make sure you have it.

**Comment by PoojaNamde** [comment]
Please share the build details once its available then we will regress this.

**Comment by merriemcgaw** [comment]
@PoojaNamde we shared a build for testing another template via email. Please use that build to regress this as well.

**Comment by kupatkar99** [comment]
#Closed; #Regressed:10-04-23;
Verified the issue in latest environment
Browser details: Microsoft Edge
Version 119.0.2132.0 (Official build) dev (64-bit)
Issue is fixed. hence closing the bug. Please find added attachment for reference.
<img width="950" alt="49790_Fixed" src="https://github.com/dotnet/aspnetcore/assets/98530069/e7c73bb6-5113-4d7e-a400-de5d20d0429d">
